public/index.js

```diff
@@ -256,13 +256,26 @@
     function addArrayToStorage(key, array){
       sessionStorage.setItem(key, JSON.stringify(array));
     }
+
+    function updateSessionStorage (key, node, array, className) {
```

Comment:
Okay this function name is weird. Something like `setLocalStorageOnClick(node, requiredClassname, key, baseArray)` would be better